streams/iq.m3u

```diff
@@ -85,3 +85,13 @@ https://5a3ed7a72ed4b.streamlock.net/zagrostv/SMIL:myStream.smil/playlist.m3u8
 https://live.karwan.tv/karwan.tv/waar-tv/playlist.m3u8
 #EXTINF:-1 tvg-id="" status="timeout",Music Al-Remas (720p) [Not 24/7]
 http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.youtube.com/c/musicalrmastv1/live
+#EXTINF:-1 tvg-id="DijlahTV.iq" status="online",Dijlah TV
+https://ghaasiflu.online/Dijlah/index.m3u8
+#EXTINF:-1 tvg-id="DijlahTarab.iq" status="online",Dijlah Tarab
+https://ghaasiflu.online/tarab/tracks-v1a1/mono.m3u8
+#EXTINF:-1 tvg-id="AlRasheedTV.iq" status="online",Al-Rasheed (408p)
+https://media1.livaat.com/AL-RASHEED-HD/tracks-v1a1/mono.m3u8
```

Comment:
```suggestion
https://media1.livaat.com/AL-RASHEED-HD/index.m3u8
```

Reply:
and quality is said to be 1080p, not 408p

Reply:
fixed